web/packages/teleport/src/components/ButtonLockedFeature/ButtonLockedFeature.tsx

```diff
@@ -29,36 +30,49 @@ export type Props = {
   [index: string]: any;
 };
 
-const salesUrl = 'https://goteleport.com/signup/enterprise/';
+const SALES_URL = 'https://goteleport.com/r/upgrade-team';
 
 export function ButtonLockedFeature({
   children,
   noIcon = false,
   event,
   ...rest
 }: Props) {
+  const ctx = useTeleport();
+  const version = ctx.storeUser.state.cluster.authVersion;
+  const isEnterprise = ctx.isEnterprise;
+
   function handleClick() {
     userEventService.captureCtaEvent(event);
-    window.open(salesUrl, 'blank');
   }
 
   return (
-    <StyledButton onClick={handleClick} {...rest}>
+    <ButtonBorder
+      as="a"
+      target="blank"
+      kind="primary"
```

Comment:
hrm we imported `ButtonBorder`, but then use kind `primary`, would just importing `ButtonPrimary` and using that work?

Reply:
We can, actually. Updated, thanks!